libraries/AP_BattMonitor/AP_BattMonitor_INA2xx.cpp

```diff
@@ -81,21 +92,27 @@ void AP_BattMonitor_INA2XX::init(void)
     if (!write_word(REG_CONFIG, REG_CONFIG_RESET) || // reset
         !write_word(REG_CONFIG, conf) ||
         !write_word(REG_CALIBRATION, cal)) {
-        GCS_SEND_TEXT(MAV_SEVERITY_WARNING, "INA2XX: Failed to configure device");
-        return;
+        return false;
     }
 
-
+    configured = true;
```

Comment:
why do we set configured before registering the callback ? 
Well, it is not like we check that the register works , but it should  be done after, shouldn't it?

Reply:
changed